packages/examples/graphiql-webpack/webpack.config.js

```diff
@@ -8,9 +8,9 @@ module.exports = {
         'react-hot-loader/patch', // activate HMR for React
         'webpack-dev-server/client?http://localhost:8080', // bundle the client for webpack-dev-server and connect to the provided endpoint
         'webpack/hot/only-dev-server', // bundle the client for hot reloading, only- means to only hot reload for successful updates
-        './index.js', // the entry point of our app
+        './index.jsx', // the entry point of our app
```

Comment:
the webpack demo was broken over use of jsx/js entrypoint, oops!

Reply:
I'm sorry; this looked like such a deliberate change I didn't vet it well enough!